cyences_app_for_splunk/default/data/ui/nav/default.xml

```diff
@@ -1,35 +1,35 @@
 <nav search_view="search">
-  <collection label="Control">
-    <view name="cs_overview" default="true" />
-    <collection label="Reports">
-      <view name="cs_ad_reports" />
-      <view name="cs_windows_reports" />
-      <view name="cs_windows_patch" />
-      <view name="cs_linux_reports" />
+  <view name="cs_overview" default="true" />
+  <view name="cs_forensics" />
+  <collection label="Reports">
```

Comment:
Shall we name it as "Dashboards" to keep it consistent with Splunk naming convention. Reports seems more like a scheduled saved search.

Reply:
Sure. I'll update that in the overview dashboard as well.